demes/demes.py

```diff
@@ -191,7 +206,9 @@ def __attrs_post_init__(self):
         # XXX: these tests should use math.isclose()
         if math.isinf(self.start_time) and self.start_size != self.end_size:
             raise ValueError("if start time is inf, must be a constant size epoch")
-        if self.size_function == "constant" and self.start_size != self.end_size:
+        if self.size_function == "constant" and not math.isclose(
```

Comment:
I think we should require them to be *exactly* equal if "constant" is the size function - it's semantically a bit weird otherwise

Reply:
Ok, I've updated to do exact equality checking (in both locations).